tests/test_angle.py

```diff
@@ -77,7 +77,7 @@ def test_invalid():
 
     # Invalid value type
     np.testing.assert_raises(TypeError, coord.Angle, theta, coord.degrees)
-    np.testing.assert_raises(ValueError, coord.Angle, 'spam', coord.degrees)
+    np.testing.assert_raises((ValueError, TypeError), coord.Angle, 'spam', coord.degrees)
```

Comment:
Also a comment here mentioning that the jax-galsim version raises a TypeError rather than a ValueError.